src/wallet/coinselection.cpp

```diff
@@ -70,7 +70,7 @@ struct {
  *
  * The algorithm uses two additional optimizations. A lookahead keeps track of the total value of
  * the unexplored UTXOs. A subtree is not explored if the lookahead indicates that the target range
- * cannot be reached. Further, it is unnecessary to test equivalent combinations (tk). This allows us
```

Comment:
In 9580c0e49030f13fe57603521b59b96e6e50cf82 "coinselection: rewrite BnB in CoinGrinder-style"

This should be squashed into 9580c0e49030f13fe57603521b59b96e6e50cf82 "coinselection: rewrite BnB in CoinGrinder-style"